generator/types.go

```diff
@@ -651,6 +657,25 @@ func (f *Field) TypeSchemaName() string {
 	return parts[len(parts)-1]
 }
 
+func (f *Field) mustFindArrayLenght() int {
+	if f.Kind != Array {
+		panic(fmt.Sprintf("%s is not an Array but a %s", f.Name, f.Type))
+	}
+
+	arrayLengthError := "kallax: It could not be possible to read the array length of %s %s\n%s"
+	found := arrayLengthRegExp.FindStringSubmatch(f.Type)
+	if len(found) < 2 {
+		panic(fmt.Sprintf(arrayLengthError, f.Name, f.Type, "No legth specified"))
+	}
```

Comment:
you can avoid all this. It's not possible that an array does not have the form `[%d]%s`